test/src/player.spec.js

```diff
@@ -446,4 +446,42 @@ describe('Track enum', function () {
   });
 });
 
+describe('events', () => {
+
+  describe('firstPlay', () => {
+    let config;
+    let player;
+
+    beforeEach(() => {
+      config = sourcesConfig.mp4_none_hls_dash;
+      player = new Player(config);
+    });
+
+    afterEach(() => {
+      player.destroy();
+    });
+
+    after(() => {
+      removeVideoElementsFromTestPage();
+    });
+
+    it('should fire first play only once', (done) => {
+      let counter = 0;
+      let onPlaying = () => {
+        player.removeEventListener(Html5Events.PLAYING, onPlaying);
+        player.pause();
+        player.play();
+        setTimeout(() => {
+          counter.should.equal(1);
```

Comment:
this is a test. i prefer to test like this. i'm appreciate your review but it should't to be 'requested changes'  !!!